www/script/templates/popups.hbs

```diff
@@ -26,16 +26,7 @@
 {{#showPendingJobs}}
     <h3>Pending Build Requests</h3>
     <ul class="pending-job-list">
-        <li>
```

Comment:
I think if we plan to put this back we might want to comment it out? 

@kkateq What do you think should we wrap this in <!--  --!>   ?